curiosrc/seal/task_trees.go

```diff
@@ -228,13 +218,19 @@ func (t *TreesTask) CanAccept(ids []harmonytask.TaskID, engine *harmonytask.Task
 }
 
 func (t *TreesTask) TypeDetails() harmonytask.TaskTypeDetails {
+	ssize := abi.SectorSize(32 << 30) // todo task details needs taskID to get correct sector size
+	if isDevnet {
+		ssize = abi.SectorSize(2 << 20)
+	}
+
 	return harmonytask.TaskTypeDetails{
 		Max:  t.max,
 		Name: "SDRTrees",
 		Cost: resources.Resources{
-			Cpu: 1,
-			Gpu: 1,
-			Ram: 8000 << 20, // todo
+			Cpu:     1,
+			Gpu:     1,
+			Ram:     8000 << 20, // todo
```

Comment:
Can we wrap this up?

Reply:
https://github.com/filecoin-project/lotus/issues/11755